lib/gtfsplus/components/GtfsPlusField.js

```diff
@@ -42,7 +42,7 @@ export default class GtfsPlusField extends Component<Props> {
   }
 
   render () {
-    const {currentValue, feedVersion, field, getGtfsEntity} = this.props
+    const {currentValue, data, feedVersion, field, getGtfsEntity} = this.props
```

Comment:
Is it possible to rename/alias this variable or add a comment? It looks like it contains the entire gtfsplus.yml but I can't fully tell...

Reply:
I totally forgot about this comment, but yes, thanks, `data` has the data for one row in the GTFS+ file. Renamed in https://github.com/ibi-group/datatools-ui/pull/763/commits/ef95f5f7579e9a7ff346b3ab520fc5297782e0d9.